pkg/controller/nodemaintenance/nodemaintenance_controller_test.go

```diff
@@ -13,28 +13,20 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/informers"
+	coordv1beta1 "k8s.io/api/coordination/v1beta1"
 
+	k8sdiscovery "k8s.io/client-go/discovery"
 	k8sfakeclient "k8s.io/client-go/kubernetes/fake"
-	"k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/client-go/rest"
+
+	k8scheme "k8s.io/client-go/kubernetes/scheme"
```

Comment:
reason?

Reply:
I think at some stage I had a import clash, so I had to rename the import package. Later I undid the change and the problem disappeared.

Reply:
Can we undo the change then please?